packages/pygments.rb

```diff
@@ -3,12 +3,13 @@
 class Pygments < Package
   description 'Python Syntax Highlighter'
   homepage 'https://pygments.org/'
-  version '2.7.3-1'
+  version '2.7.3-2'
   compatibility 'all'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
   def self.install
-    system "pip3 install pygments --no-warn-script-location --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
+    system "pip3 uninstall -y pygments"
```

Comment:
What happens if pygments isn't installed?

Reply:
Still works:
```
chronos@localhost ~/src/satadru/chromebrew $ crew remove pygments
Pygments removed!
chronos@localhost ~/src/satadru/chromebrew $ crew install pygments
pygments: Python Syntax Highlighter
https://pygments.org/
version 2.7.3-4
No precompiled binary available for your platform, downloading source...
Archive found in cache
gcc (GCC) 10.2.1 20201217
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
Preconfiguring package...
WARNING: Skipping pygments as it is not installed.
Collecting pygments
  Using cached Pygments-2.7.3-py3-none-any.whl (950 kB)
Installing collected packages: pygments
Successfully installed pygments-2.7.3
Rename all *.la_tmp files back to *.la
Performing pre-install...
Performing install...
Performing post-install...
Pygments installed!
```
Also
```
chronos@localhost ~/src/satadru/chromebrew $ crew remove pygments
Pygments removed!
chronos@localhost ~/src/satadru/chromebrew $ crew upgrade pygments
pygments: Python Syntax Highlighter
https://pygments.org/
version 2.7.3-4
Updating pygments...
No precompiled binary available for your platform, downloading source...
Archive found in cache
gcc (GCC) 10.2.1 20201217
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
Preconfiguring package...
WARNING: Skipping pygments as it is not installed.
Collecting pygments
  Using cached Pygments-2.7.3-py3-none-any.whl (950 kB)
Installing collected packages: pygments
Successfully installed pygments-2.7.3
Rename all *.la_tmp files back to *.la
Removing since upgrade or reinstall...
Package pygments isn't installed.
Performing pre-install...
Performing install...
Performing post-install...
Pygments installed!
```